src/toil/provisioners/abstractProvisioner.py

```diff
@@ -52,7 +46,7 @@ class AbstractProvisioner(object):
     __metaclass__ = ABCMeta
 
 
-    def __init__(self, config=None, batchSystem=None):
+    def __init__(self, config=None):
         """
```

Comment:
Why doesn't this take in batchSystem as an argument anymore? Also the docstring below needs to be changed to reflect that. I realize this has already been merged, but I'm hoping @cket can help explain this design choice.